package.json

```diff
@@ -115,6 +115,7 @@
         "shasum": "1.0.2",
         "systeminformation": "5.17.12",
         "tailwindcss": "3.3.2",
+        "tree-kill": "^1.2.2",
```

Comment:
`tree-kill` has not been updates in 5 years, nov 2018, and should probably not be used.
There is a different project called `tree-kill-promise`, which is more recently updated: https://github.com/dword-design/tree-kill-promise.

Tree-kill-promise:
https://github.com/dword-design/tree-kill-promise
Distributed package seems well done 🤘
https://unpkg.com/browse/tree-kill-promise@3.0.14/